app/views/estimatedIncomeTax/NoCurrentIncomeView.scala.html

```diff
@@ -43,18 +43,25 @@
 
 @header = @{
     ptaHeader(
-        displayBackLink = true,
+        displayBackLink = false,
         mainHeadingText = Messages("tai.estimatedIncome.title"),
         preHeadingText = taxYearRangeString,
-        preHeadingAccessibleAnnouncement = Messages("tai.estimatedIncome.accessiblePreHeading")
+        preHeadingAccessibleAnnouncement = Messages("tai.estimatedIncome.accessiblePreHeading"),
+        showGdsVersion = true
```

Comment:
How did you know to add this?

Reply:
It's the only one that will show: 
```
<span class="govuk-caption-xl"><span class="govuk-visually-hidden">@preHeadingHidden </span>@Html(preHeadingText)</span>
    <h1 class="govuk-heading-xl heading-mobile-responsive xbreak-word">@Html(mainHeadingText)</h1>
    ```

Which is necessary for the accessibility stuff. I think this if/else can be removed once we finish with the library upgrades...